### PR TITLE
feat(border): Add width and radius tokens

### PR DESCRIPTION
**Pull request by alexandra-lim**
This MR adds border width and radius tokens to the `primitives.json` file.

### CODE REVIEW

**Comment by CCAyl** [comment]
is this ready for review? i'm so sorry i've completely ignored this!

**Comment by CCAyl** [comment]
This LGTM!

**Review by CCAyl — APPROVED** [review]

**Review by iryanclarke — APPROVED** [review]
Very noice

**Comment by iryanclarke** [comment]
Oh womp, the new yellow colors cause bigly merge conflicts

**Comment by alexandra-lim** [comment]
Anna gave me the new border radius - from 4px to 8px so will fix the conflicts once I add those :)

File: src/data/primitives.json [hunk 1]
```diff
@@ -1,4 +1,34 @@
 {
+  "borders": {
+    "radius": {
+      "base": {
+        "value": "8px",
+        "type": "borderRadius",
+        "description": "Default border radius option for most components including card-type components like NewBox, Accordion, etc."
+      },
```
Comment (alexandra-lim) [inline src/data/primitives.json:8]: Removed the card version and updated base to be 8px from 4px.

Comment (CCAyl) [reply src/data/primitives.json:8]: should we just change this to say `Box` ? under the assumption we are going to remove the `New` prefix? just one less thing to update later 🍅 🍅

Comment (alexandra-lim) [reply src/data/primitives.json:8]: Yeah great call out! Done :)

**Review by CCAyl — APPROVED** [review]
shes perfect. so glad we can use this for our ds4 changes!

**Pull request merged**